### ISSUE TITLE
TypeError: undefined is not an object (evaluating 'this.setState')

### ISSUE THREAD

**Issue by oxymor0n** [getsentry/sentry#3020]
When I tried to change a team name in my on-prem Sentry 8.3.0, this error came up and got logged in Sentry/Internal. The stack trace is found below

```
TypeError: undefined is not an object (evaluating 'this.setState')
  at call(./app/components/forms/inputField.jsx:38:4)
  at executeOnChange(~/react/lib/LinkedValueUtils.js:129:0)
  at func(~/react/lib/ReactDOMInput.js:106:0)
  at o([native code])
  at invokeGuardedCallback(~/react/lib/ReactErrorUtils.js:27:0)
  at executeDispatch(~/react/lib/EventPluginUtils.js:79:0)
  at executeDispatchesInOrder(~/react/lib/EventPluginUtils.js:102:0)
  at executeDispatchesAndRelease(~/react/lib/EventPluginHub.js:43:0)
  at forEach(~/react/lib/EventPluginHub.js:54:0)
  at forEach([native code])
  at forEachAccumulated(~/react/lib/forEachAccumulated.js:23:0)
  at processEventQueue(~/react/lib/EventPluginHub.js:259:0)
  at runEventQueueInBatch(~/react/lib/ReactEventEmitterMixin.js:18:0)
  at _handleTopLevel(~/react/lib/ReactEventEmitterMixin.js:34:0)
  at handleTopLevelWithoutPath(~/react/lib/ReactEventListener.js:93:0)
  at call(~/react/lib/ReactEventListener.js:73:0)
  at perform(~/react/lib/Transaction.js:136:0)
  at batchedUpdates(~/react/lib/ReactDefaultBatchingStrategy.js:62:0)
  at batchedUpdates(~/react/lib/ReactUpdates.js:94:0)
  at apply(~/react/lib/ReactEventListener.js:204:0)
  at ? ([native code])
  at apply(~/raven-js/src/raven.js:250:0)
```

**Comment by mattrobenolt** [comment]
@benvinegar is this what was fixed for 8.3.1?

**Comment by mattrobenolt** [comment]
Seems related to the inputField issue, which affected saved searches. I assume related?

**Comment by benvinegar** [comment]
@mattrobenolt Yes.

**Comment by mattrobenolt** [comment]
@oxymor0n then this has been fixed in 8.3.1. See: https://github.com/getsentry/sentry/pull/2985

**Comment by oxymor0n** [comment]
thank you all for the quick reply
